src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1292,9 +1309,6 @@ def _process_cls_names(info_dict, names, write_doc=False):
             cls.child_object_type = get_cls(
                 "child-object-type", object_type, cls, version=version
             )
-            cls.child_object_type.rename = lambda self, name: self._parent.rename(
-                name, self._name
-            )
```

Comment:
This was added in server side to maintain backwards compatibility. However, it is not required as the backend settings-api framework has not changed and thus it handles the earlier cases as well.
The server-side code will also be updated accordingly.

Thank you @gyeole and @V-Dandekar for your inputs.